### ISSUE TITLE
KDE plot behaves inconsistently with and without statsmodels

### ISSUE THREAD

**Issue by jwdebelius** [mwaskom/seaborn#796]
I'm using KDE plots to generate a kernel smoothed plot in Seaborn 0.6.0. I used the following code to generate a plot with a vector of data:

``` python
import numpy as np
import seaborn as sn
import matplotlib.pyplot as plt

sn.set_style('ticks', {'axes.facecolor': 'none'})
ax = plt.axes()
sn.kdeplot(data, ax=ax, legend=False)
sn.kdeplot(data, ax=ax, legend=False, clip=[5, 10])
```

The first time I used the function, I had a conda environment with seaborn installed, but no statsmodels. It rendered a plot where clip limited the shaded area in the second plot, but the shaded area had the same kernel as the parent function. This is what I'd like.

![no_statsmodels](https://cloud.githubusercontent.com/assets/5534846/11707721/d282601c-9ebc-11e5-82a2-5f96e10e98a7.png)

I then conda installed statsmodels (the latest version via conda install) because I wanted to use some of that functionality in that package. With statsmodels in the environment, the kernel smoothing on my clipped area changed, and it fit the data for only that subset, creating a new smoothed kernel.

![statsmodels](https://cloud.githubusercontent.com/assets/5534846/11707622/b94610ea-9ebb-11e5-8cce-53e23840e190.png)

It would be really nice to have a consistent behavior. For my application, shading a clipped area of the same plot is handy.

**Comment by mwaskom** [comment]
FWIW the second behavior is probably the "right" one, so these could be made more consistent, but I'm not sure it would help you.

**Comment by mwaskom** [comment]
A robust, but more verbose, approach would be

```

**Comment by JWDebelius** [comment]
Thanks!

**Comment by mwaskom** [comment]
Closed in #2104 with the removal of the statsmodels backend
